.github/workflows/makefile.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: "1.16.x"
+          go-version: "1.19.x"
```

Comment:
`quic-go` requires go version >1.18.0, thus the `makefile.yml` CI job fails with `go-version: "1.16.x"`. I am not familiar with workflows for golang; is it enough to just update the version here?